storage/errors.go

```diff
@@ -0,0 +1,14 @@
+package storage
+
+import (
+	"fmt"
+)
+
+// ItemNotFoundError shows that item with id ItemID wasn't found in the storage
+type ItemNotFoundError struct {
+	ItemID interface{}
+}
+
+func (e *ItemNotFoundError) Error() string {
```

Comment:
please add docstring (it's straightforward there)

Reply:
fixed